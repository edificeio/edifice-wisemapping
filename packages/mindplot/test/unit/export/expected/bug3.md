# Indicator needs

- Which new measures[^1] 
	- Landscape of measures
		- Diversity index of innovation support instruments in the region[^2] 
		- Existing investments in measures
	- What other regions do differently
		- Balance of measure index
		- Profile comparison with other regions
		- Number of specific types of measures per capita
- How to design & implement measures[^3] 
	- Good practices
	- Diagnostics
		- Internal business innovation factors
		- Return on investment to innovation
			- Firm's turnover from (new to firm)
 product innovation (as a pecentage of total turnover)
			- Increase in the probability to innovate linked to ICT use
 (in product innovation, process innovation, organisational innovaton, marketing innovation)
			- Scientific articles by type of collaboration (per capita)
 (international co-authoriship, domestic co-authoriship, single author)
			- Increase in a share of expenditures on technological
innovations in the total amount of regional firmsâ€™ expenditures, %
			- Increase in the number of innovative companies with in-house R&D
			- Increase in th number of innovative companies without in-house R&D
			- Increase in th number of firms with
 international/national collaboration on innovation
			- Highly cited scientific articles (as a percentage of
highly cited scientific article in the whole Federation)
			- Patents filed by public research organisations
(as a percentafe of patent application filed under PCT)
			- Number of international patents
			- Start-up activity (as a percentage of start-up activity in the whole Federation)
			- Number of innovative companies to the number of students 
			- Number of innovative companies to the number of researchers  
			- Volume of license agreements to the volume of R&D support from the regional budget 
- How much effort: where & how[^4] 
	- The bottom-line[^5] 
		- Wages
			- Dynamics of real wages
			- Average wage (compare to the Fed)
		- Productivity
			- Labor productivity
			- Labor productivity growth rate
		- Jobs
			- Share of high-productive jobs
			- Share of creative industries jobs
			- Uneployment rate of university graduates
		- Income
			- GRP per capita and its growth rate
	- Influencing  factors
		- Economy
			- Economic structure
			- Volume of manufacturing production per capita 
			- Manufacturing value added per capita (non-natural resource-based)
		- The enabling environment
			- Ease of doing business[^6] 
				- Level of administrative barriers (number and cost of administrative procedures) 
			- Competition index[^7] 
			- Workforce
				- Quality of education[^8] 
					- Inrease in the number of International students
				- Quantity of education
					- Participation in life-long learning[^9] 
					- Increase in literarecy 
					- Amount of university and colleague
students per 10 thousands population
					- Share of employees with higher education in
the total amount of population at the working age
					- Increase in University students
					- Government expenditure on General University Funding
					- Access to training, information, and consulting support 
				- Science & engineering workforce
					- Availability of scientists and engineers[^10] 
					- Amount of researches per 10 thousands population
					- Average wage of researches per average wage in the region
					- Share of researchers in the total number of employees in the region
			- Government
				- Total expenditure of general government as a percentage of GDP
				- Government expenditure on Economic Development
			- Access to finance
				- Deals
					- Venture capital investments for start-ups as a percentage of GDP
					- Amounts of business angel, pre-seed, seed and venture financing
					- Amount of public co-funding of business R&D
					- Number of startups received venture financing 
					- Number of companies received equity investments 
				- Available
					- Amount of matching grants available in the region for business R&D
					- Number of Business Angels
			- ICT
				- ICT use[^11] 
				- Broadband penetration 
				- Internet penetration
				- Computer literacy 
		- Behavior of innovation actors
			- Access to markets
				- FDI
					- foreign JVs
					- Inflow of foreign direct investments in high-technology industries
					- Foreign direct investment jobs[^12] 
					- FDI as a share of regional non natural resource-based GRP 
					- Number of foreign subsidiaries operating in the region
					- Share of foreign controlled enterprises
				- Exports
					- Export intensity in manufacturing and services[^13] 
					-  Share of high-technology export in the total volume
 of production of goods, works and services
					- Share of innovation production/serivces that goes for export,
by zones (EU, US, CIS, other countries
				- Share of high-technology products in government procurements
			- Entrepreneurship culture
				- Fear of failure rate[^14] 
				- Entrepreneurship as desirable career choice[^15] 
				- High Status Successful Entrepreneurship[^16] 
			- Collaboration & partnerships
				- Number of business contracts with foreign partners for R&D collaboration
				- Share of R&D financed from foreign sources[^17] 
				- Firms collaborating on innovation with organizations in other countries[^18] 
				- Share of Innovative companies collaborating
with research institutions on innovation
				- Number of joint projects conducted by the local comapnies
 and local consulting/intermediary agencies
				- science and industry links
			- Technology absorption
				- Local supplier quality[^19] 
				- Share of expenditures on technological innovations
in the amount of sales
				- Number of purchased new technologies
				- Investments in ICT by asset (IT equipment,
 communication equipment, software)
				- Machinery and equipment
				- Software and databases
				- Level of energy efficiency of the regional economy
(can be measured by sectors and for the whole region)
				- Share of wastes in the total volume of production (by sector)
			- Innovation activities in firms
				- Share of innovative companies
				- Business R&D expenditures per GRP
				- Factors hampering innovation[^20] 
				- Expenditure on innovation by firm size
				- R&D and other intellectl property products
				- Growth of the number of innovative companies 
				- Outpus
					- Volume of new to Russian market production per GRP
					- Volume of new to world market production per total production
					- Growth of the volume of production of innovative companies 
					- Volume of innovation production per capita 
			- Entrepreneurial activities
				- New business density[^21] 
				- Volume of newly registered corporations [^22] 
				- Share of gazelle companies in the total number of businesses
			- R&D production
				- Outputs
					- Amount of domestically protected intellectual
 property per 1 mln. population
					- Amount of PCT-applications per 1 mln. population
					- Number of domestic patent applications per R&D expenditures
					- Number of intellectual property exploited by regional
enterprises per 1 mln. population
					- Publication activity of regional scientists and researches
				- Inputs
					- Regional and local budget expenditures on R&D
					- Government R&D expenditure 
			- Public sector innovation
				- Number of advanced ICT introduced in the budgetary organizations
(regional power, municipal bodies, social and educational organizations)	
				- E-government index
				- Number of management innovations introduced in the budgetary organizations
 (regional power, municipal bodies, social and educational organizations)	
		- Supporting organizations
			- Research institutions
				- Collaboration
					- Number of interactions between universities
and large companies  by university size
				- Resources
					- R&D expenditures per 1 researcher
					- Average wage of researches per average wage in the region
					- High education expenditure on R&D
				- Scientific outputs
					- Publications
						- Impact of publications in the ISI database (h-index)
						- Number of publications in international journals per worker per year
						- Publications: Academic articles in international peer-reviewed
 journals per 1,000 researchers [articles/1,000 researchers].
					- Number of foreign patents granted per staff
				- Supportive measures
					- Diversity index of university entrepreneurship support measures[^23] 
				- Commercialization
					- Licensing
						- Academic licenses: Number of licenses
per 1,000 researchers.[licenses/researcher] 
					- Spin-offs
						- Number of spin-offs with external private financing
as a share of the institution's R&D budget
					- Industry contracts
						- Industry revenue per staff 
						- Foreign contracts: Number of contracts with foreign industria
l companies at scientific and educational organizations
per 1,000 researchers [contracts/researchers]
						- Share of industry income from foreign companies
						- Revenue raised from industry R&D as a fraction
of total institutional budget (up to a cap)
						- Difficulties faced by research organization in collaborating with SMEs
			- Private market
				- Number of innovation & IP services organizations[^24] 
				- Number of private innovation infrastructure organizations [^25] 
				- Access to certification and licensing for specific activities 
				- Access to suppliers of equipment, production and engineering services 
			- Innovation infrastructure
				- Investments
					- Public investment in innovation infrastructure
					- Increase of government investment in innovation infrastructure
					-  Number of Development institution projects performed in the region
					- Volume of seed investments by the regional budget 
					- Volume of venture financing from the regional budget 
				- Volume of state support per one company 
- What to do about existing measures[^26] 
	- Demand for measure
		- Quality of beneficiaries
			- Growth rates of employment in supported innovative firms
			- Growth rates of employment in supported innovative firms
			- Role of IP for tenants/clients[^27] 
			- Share of tenants with innovation activities
			- Gazelle tenant: Share of tenants with
annual revenue growth of more than 20%
 for each of the past four years or since formation [%]
			- Globalization of tenants: Median share of tenant
revenues obtained from exports [%]
		- Number of beneficiaries
			- Number of projects conducted by companies in cooperation with innovation infrastructure
			- Scope and intensity of use of services offered to firms
			- Number of companies supported by the infrastructure (training, information, consultations, etc.)
			- Increase in the number of business applying for public support programmes (regional, federal, international) 
		- Degree of access
			- Level of awareness
				- Perception (opinion poll) of business managers
 regarding public support programmes
			- Transparency
				- Perception of business managers in terms
 of level of transparency of support measures in the region
			- Description by regional business managers of the way the
select and apply for regional and federal support schemes
		- Number of applicants
			- Increase in the number of business applying for public support programmes
			- Number of companies that know about a particular program
			- Increase in the number of start-ups applying to receive VC investments
			- Increase in the number of start-ups applying for a place in the incubators
	- Inputs of measures
		- Qualified staff[^28] 
		- Budget per beneficiary
	- Performance of measure
		- Implementation of measure
			- Target vs. actual KPIs
			- Intermediate outputs per budget
			- Qualification of staff
		- Output of measure
			- Opinion surveys
				- Opinions of beneficiaries
			- Hard metrics
				- Output per headcount (e.g. staff, researchers)
				- Productivity analysis
	- Impact of measure
		- Opinion surveys
			- Perception of support impact (opinion polls)
			- Perception of the activity of regional government by the regional companies 
		- Hard metrics
			- Increase in number of small innovation enterprises 
			- Growth of the total volume of salary in the supported companies (excluding inflation) 
			- Growth of the volume of regional taxes paid by the supported companies 
			- Growth of the volume of export at the supported companies 
			- Number of new products/projects at the companies that received support 
		- Impact assessment 
		- Average leverage of 1rub (there would be
several programs with different leverage)
		- Volume of attracted money per one ruble
of regional budget expenditures on innovation projects
- What investments in innovative projects[^29] 
	- Competitive niches
		- Clusters behavior
			- Cluster EU star rating
			- Share of value added of cluster enterprises in GRP
			- Share of cluster products in the relevant world market segment 
			- Share of export in cluster total volume of sales
			- Growth of the volume of production in the cluster companies
			- Growth of the volume of production in the cluster companies
 to the volume of state support for the cluster
			- Growth of the volume of innovation production in the cluster
			- Share of export in cluster total volume of sales (by zones: US, EU, CIS, other countries) 
			- Internal behavior
				- Median wage in the cluster
				- Growth of the volume of R&D in the cluster
				- Cluster collaboration
		- R&D
			- Patent map
			- Publications map
		- Industry
			- FDI map
			- Gazelle map
			- Business R&D expenditures as a share of revenues by sector
			- Share of regional products in the world market
			- Expenditure on innovation by firm size, by sector 
		- Entrepreneurship
			- Startup map
			- Venture investment map
			- Attractiveness to public competitive funding
				- Fed and regional seed fund investments
				- FASIE projects: Number of projects supported
by the FASIE per 1,000 workers [awards/worker] 
	- Competitiveness support factors
		- Private investment in innovation
- How to improve image
	- Rankings
		- macro indicators
		- meso-indicators
	- Innovation investment climate



[^1]: Identifying new measures or investments that should be implemented.[^2]: Number of different innovations policy instruments existing  in the region as a share of a total number representing a full typology of instruments[^3]: Understanding how to design the details of a particular measure and how to implement them.[^4]: Understanding the level of effort the region needs to take to compete on innovation and where to put this effort[^5]: This is what policy makers care about in the end[^6]: WB[^7]: GCR[^8]: GCR[^9]: per 100 population aged 25-64[^10]: GCR[^11]: GCR[^12]: : the percentage of the workforce employed by foreign companies [%]. [^13]: : exports as a share of total output in manufacturing and services [%].[^14]: GEM[^15]: GEM[^16]: GEM[^17]: UNESCO[^18]: CIS[^19]: GCR[^20]: CIS, BEEPS[^21]: Number of new organizations per thousand working age population (WBI)[^22]: (as a percentage of all registered corporations)[^23]: Number of measures offered by the unversity within a preset range (NCET2 survey)[^24]:  (design firms, IP consultants, etc.)[^25]: (e.g. accelerators, incubators)[^26]: Understanding which measures should be strengthened, dropped or improved, and how.[^27]: WIPO SURVEY OF INTELLECTUAL PROPERTY SERVICES OFEUROPEAN TECHNOLOGY INCUBATORS[^28]: JL: not sure how this would be measured[^29]: Understanding what investments should be made in innovative projects.
