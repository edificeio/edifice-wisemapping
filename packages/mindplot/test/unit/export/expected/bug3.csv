Indicator needs 
,
            Which new measures 
,
            ,
            Landscape of measures 
,
            ,
            ,
            Diversity index of innovation support instruments in the region 
,
            ,
            ,
            Existing investments in measures 
,
            ,
            What other regions do differently 
,
            ,
            ,
            Balance of measure index 
,
            ,
            ,
            Profile comparison with other regions 
,
            ,
            ,
            Number of specific types of measures per capita 
,
            How to design & implement measures 
,
            ,
            Good practices 
,
            ,
            Diagnostics 
,
            ,
            ,
            Internal business innovation factors 
,
            ,
            ,
            Return on investment to innovation 
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            Increase in the number of innovative companies with in-house R&D 
,
            ,
            ,
            ,
            Increase in th number of innovative companies without in-house R&D 
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            Number of international patents 
,
            ,
            ,
            ,
            Start-up activity (as a percentage of start-up activity in the whole Federation) 
,
            ,
            ,
            ,
            Number of innovative companies to the number of students  
,
            ,
            ,
            ,
            Number of innovative companies to the number of researchers   
,
            ,
            ,
            ,
            Volume of license agreements to the volume of R&D support from the regional budget  
,
            How much effort: where & how 
,
            ,
            The bottom-line 
,
            ,
            ,
            Wages 
,
            ,
            ,
            ,
            Dynamics of real wages 
,
            ,
            ,
            ,
            Average wage (compare to the Fed) 
,
            ,
            ,
            Productivity 
,
            ,
            ,
            ,
            Labor productivity 
,
            ,
            ,
            ,
            Labor productivity growth rate 
,
            ,
            ,
            Jobs 
,
            ,
            ,
            ,
            Share of high-productive jobs 
,
            ,
            ,
            ,
            Share of creative industries jobs 
,
            ,
            ,
            ,
            Uneployment rate of university graduates 
,
            ,
            ,
            Income 
,
            ,
            ,
            ,
            GRP per capita and its growth rate 
,
            ,
            Influencing  factors 
,
            ,
            ,
            Economy 
,
            ,
            ,
            ,
            Economic structure 
,
            ,
            ,
            ,
            Volume of manufacturing production per capita  
,
            ,
            ,
            ,
            Manufacturing value added per capita (non-natural resource-based) 
,
            ,
            ,
            The enabling environment 
,
            ,
            ,
            ,
            Ease of doing business 
,
            ,
            ,
            ,
            ,
            Level of administrative barriers (number and cost of administrative procedures)  
,
            ,
            ,
            ,
            Competition index 
,
            ,
            ,
            ,
            Workforce 
,
            ,
            ,
            ,
            ,
            Quality of education 
,
            ,
            ,
            ,
            ,
            ,
            Inrease in the number of International students 
,
            ,
            ,
            ,
            ,
            Quantity of education 
,
            ,
            ,
            ,
            ,
            ,
            Participation in life-long learning 
,
            ,
            ,
            ,
            ,
            ,
            Increase in literarecy  
,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
            Increase in University students 
,
            ,
            ,
            ,
            ,
            ,
            Government expenditure on General University Funding 
,
            ,
            ,
            ,
            ,
            ,
            Access to training, information, and consulting support  
,
            ,
            ,
            ,
            ,
            Science & engineering workforce 
,
            ,
            ,
            ,
            ,
            ,
            Availability of scientists and engineers 
,
            ,
            ,
            ,
            ,
            ,
            Amount of researches per 10 thousands population 
,
            ,
            ,
            ,
            ,
            ,
            Average wage of researches per average wage in the region 
,
            ,
            ,
            ,
            ,
            ,
            Share of researchers in the total number of employees in the region 
,
            ,
            ,
            ,
            Government 
,
            ,
            ,
            ,
            ,
            Total expenditure of general government as a percentage of GDP 
,
            ,
            ,
            ,
            ,
            Government expenditure on Economic Development 
,
            ,
            ,
            ,
            Access to finance 
,
            ,
            ,
            ,
            ,
            Deals 
,
            ,
            ,
            ,
            ,
            ,
            Venture capital investments for start-ups as a percentage of GDP 
,
            ,
            ,
            ,
            ,
            ,
            Amounts of business angel, pre-seed, seed and venture financing 
,
            ,
            ,
            ,
            ,
            ,
            Amount of public co-funding of business R&D 
,
            ,
            ,
            ,
            ,
            ,
            Number of startups received venture financing  
,
            ,
            ,
            ,
            ,
            ,
            Number of companies received equity investments  
,
            ,
            ,
            ,
            ,
            Available 
,
            ,
            ,
            ,
            ,
            ,
            Amount of matching grants available in the region for business R&D 
,
            ,
            ,
            ,
            ,
            ,
            Number of Business Angels 
,
            ,
            ,
            ,
            ICT 
,
            ,
            ,
            ,
            ,
            ICT use 
,
            ,
            ,
            ,
            ,
            Broadband penetration  
,
            ,
            ,
            ,
            ,
            Internet penetration 
,
            ,
            ,
            ,
            ,
            Computer literacy  
,
            ,
            ,
            Behavior of innovation actors 
,
            ,
            ,
            ,
            Access to markets 
,
            ,
            ,
            ,
            ,
            FDI 
,
            ,
            ,
            ,
            ,
            ,
            foreign JVs 
,
            ,
            ,
            ,
            ,
            ,
            Inflow of foreign direct investments in high-technology industries 
,
            ,
            ,
            ,
            ,
            ,
            Foreign direct investment jobs 
,
            ,
            ,
            ,
            ,
            ,
            FDI as a share of regional non natural resource-based GRP  
,
            ,
            ,
            ,
            ,
            ,
            Number of foreign subsidiaries operating in the region 
,
            ,
            ,
            ,
            ,
            ,
            Share of foreign controlled enterprises 
,
            ,
            ,
            ,
            ,
            Exports 
,
            ,
            ,
            ,
            ,
            ,
            Export intensity in manufacturing and services 
,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            Share of high-technology products in government procurements 
,
            ,
            ,
            ,
            Entrepreneurship culture 
,
            ,
            ,
            ,
            ,
            Fear of failure rate 
,
            ,
            ,
            ,
            ,
            Entrepreneurship as desirable career choice 
,
            ,
            ,
            ,
            ,
            High Status Successful Entrepreneurship 
,
            ,
            ,
            ,
            Collaboration & partnerships 
,
            ,
            ,
            ,
            ,
            Number of business contracts with foreign partners for R&D collaboration 
,
            ,
            ,
            ,
            ,
            Share of R&D financed from foreign sources 
,
            ,
            ,
            ,
            ,
            Firms collaborating on innovation with organizations in other countries 
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            science and industry links 
,
            ,
            ,
            ,
            Technology absorption 
,
            ,
            ,
            ,
            ,
            Local supplier quality 
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            Number of purchased new technologies 
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            Machinery and equipment 
,
            ,
            ,
            ,
            ,
            Software and databases 
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            Share of wastes in the total volume of production (by sector) 
,
            ,
            ,
            ,
            Innovation activities in firms 
,
            ,
            ,
            ,
            ,
            Share of innovative companies 
,
            ,
            ,
            ,
            ,
            Business R&D expenditures per GRP 
,
            ,
            ,
            ,
            ,
            Factors hampering innovation 
,
            ,
            ,
            ,
            ,
            Expenditure on innovation by firm size 
,
            ,
            ,
            ,
            ,
            R&D and other intellectl property products 
,
            ,
            ,
            ,
            ,
            Growth of the number of innovative companies  
,
            ,
            ,
            ,
            ,
            Outpus 
,
            ,
            ,
            ,
            ,
            ,
            Volume of new to Russian market production per GRP 
,
            ,
            ,
            ,
            ,
            ,
            Volume of new to world market production per total production 
,
            ,
            ,
            ,
            ,
            ,
            Growth of the volume of production of innovative companies  
,
            ,
            ,
            ,
            ,
            ,
            Volume of innovation production per capita  
,
            ,
            ,
            ,
            Entrepreneurial activities 
,
            ,
            ,
            ,
            ,
            New business density 
,
            ,
            ,
            ,
            ,
            Volume of newly registered corporations  
,
            ,
            ,
            ,
            ,
            Share of gazelle companies in the total number of businesses 
,
            ,
            ,
            ,
            R&D production 
,
            ,
            ,
            ,
            ,
            Outputs 
,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
            Amount of PCT-applications per 1 mln. population 
,
            ,
            ,
            ,
            ,
            ,
            Number of domestic patent applications per R&D expenditures 
,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
            Publication activity of regional scientists and researches 
,
            ,
            ,
            ,
            ,
            Inputs 
,
            ,
            ,
            ,
            ,
            ,
            Regional and local budget expenditures on R&D 
,
            ,
            ,
            ,
            ,
            ,
            Government R&D expenditure  
,
            ,
            ,
            ,
            Public sector innovation 
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            E-government index 
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            Supporting organizations 
,
            ,
            ,
            ,
            Research institutions 
,
            ,
            ,
            ,
            ,
            Collaboration 
,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            Resources 
,
            ,
            ,
            ,
            ,
            ,
            R&D expenditures per 1 researcher 
,
            ,
            ,
            ,
            ,
            ,
            Average wage of researches per average wage in the region 
,
            ,
            ,
            ,
            ,
            ,
            High education expenditure on R&D 
,
            ,
            ,
            ,
            ,
            Scientific outputs 
,
            ,
            ,
            ,
            ,
            ,
            Publications 
,
            ,
            ,
            ,
            ,
            ,
            ,
            Impact of publications in the ISI database (h-index) 
,
            ,
            ,
            ,
            ,
            ,
            ,
            Number of publications in international journals per worker per year 
,
            ,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
            Number of foreign patents granted per staff 
,
            ,
            ,
            ,
            ,
            Supportive measures 
,
            ,
            ,
            ,
            ,
            ,
            Diversity index of university entrepreneurship support measures 
,
            ,
            ,
            ,
            ,
            Commercialization 
,
            ,
            ,
            ,
            ,
            ,
            Licensing 
,
            ,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
            Spin-offs 
,
            ,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
            Industry contracts 
,
            ,
            ,
            ,
            ,
            ,
            ,
            Industry revenue per staff  
,
            ,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
            ,
            Share of industry income from foreign companies 
,
            ,
            ,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            ,
            ,
            ,
            Difficulties faced by research organization in collaborating with SMEs 
,
            ,
            ,
            ,
            Private market 
,
            ,
            ,
            ,
            ,
            Number of innovation & IP services organizations 
,
            ,
            ,
            ,
            ,
            Number of private innovation infrastructure organizations  
,
            ,
            ,
            ,
            ,
            Access to certification and licensing for specific activities  
,
            ,
            ,
            ,
            ,
            Access to suppliers of equipment, production and engineering services  
,
            ,
            ,
            ,
            Innovation infrastructure 
,
            ,
            ,
            ,
            ,
            Investments 
,
            ,
            ,
            ,
            ,
            ,
            Public investment in innovation infrastructure 
,
            ,
            ,
            ,
            ,
            ,
            Increase of government investment in innovation infrastructure 
,
            ,
            ,
            ,
            ,
            ,
             Number of Development institution projects performed in the region 
,
            ,
            ,
            ,
            ,
            ,
            Volume of seed investments by the regional budget  
,
            ,
            ,
            ,
            ,
            ,
            Volume of venture financing from the regional budget  
,
            ,
            ,
            ,
            ,
            Volume of state support per one company  
,
            What to do about existing measures 
,
            ,
            Demand for measure 
,
            ,
            ,
            Quality of beneficiaries 
,
            ,
            ,
            ,
            Growth rates of employment in supported innovative firms 
,
            ,
            ,
            ,
            Growth rates of employment in supported innovative firms 
,
            ,
            ,
            ,
            Role of IP for tenants/clients 
,
            ,
            ,
            ,
            Share of tenants with innovation activities 
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
             
,
            ,
            ,
            Number of beneficiaries 
,
            ,
            ,
            ,
            Number of projects conducted by companies in cooperation with innovation infrastructure 
,
            ,
            ,
            ,
            Scope and intensity of use of services offered to firms 
,
            ,
            ,
            ,
            Number of companies supported by the infrastructure (training, information, consultations, etc.) 
,
            ,
            ,
            ,
            Increase in the number of business applying for public support programmes (regional, federal, international)  
,
            ,
            ,
            Degree of access 
,
            ,
            ,
            ,
            Level of awareness 
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            Transparency 
,
            ,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
             
,
            ,
            ,
            Number of applicants 
,
            ,
            ,
            ,
            Increase in the number of business applying for public support programmes 
,
            ,
            ,
            ,
            Number of companies that know about a particular program 
,
            ,
            ,
            ,
            Increase in the number of start-ups applying to receive VC investments 
,
            ,
            ,
            ,
            Increase in the number of start-ups applying for a place in the incubators 
,
            ,
            Inputs of measures 
,
            ,
            ,
            Qualified staff 
,
            ,
            ,
            Budget per beneficiary 
,
            ,
            Performance of measure 
,
            ,
            ,
            Implementation of measure 
,
            ,
            ,
            ,
            Target vs. actual KPIs 
,
            ,
            ,
            ,
            Intermediate outputs per budget 
,
            ,
            ,
            ,
            Qualification of staff 
,
            ,
            ,
            Output of measure 
,
            ,
            ,
            ,
            Opinion surveys 
,
            ,
            ,
            ,
            ,
            Opinions of beneficiaries 
,
            ,
            ,
            ,
            Hard metrics 
,
            ,
            ,
            ,
            ,
            Output per headcount (e.g. staff, researchers) 
,
            ,
            ,
            ,
            ,
            Productivity analysis 
,
            ,
            Impact of measure 
,
            ,
            ,
            Opinion surveys 
,
            ,
            ,
            ,
            Perception of support impact (opinion polls) 
,
            ,
            ,
            ,
            Perception of the activity of regional government by the regional companies  
,
            ,
            ,
            Hard metrics 
,
            ,
            ,
            ,
            Increase in number of small innovation enterprises  
,
            ,
            ,
            ,
            Growth of the total volume of salary in the supported companies (excluding inflation)  
,
            ,
            ,
            ,
            Growth of the volume of regional taxes paid by the supported companies  
,
            ,
            ,
            ,
            Growth of the volume of export at the supported companies  
,
            ,
            ,
            ,
            Number of new products/projects at the companies that received support  
,
            ,
            ,
            Impact assessment  
,
            ,
            ,
             
,
            ,
            ,
             
,
            How to improve image 
,
            ,
            Rankings 
,
            ,
            ,
            macro indicators 
,
            ,
            ,
            meso-indicators 
,
            ,
            Innovation investment climate 
,
            What investments in innovative projects 
,
            ,
            Competitive niches 
,
            ,
            ,
            Clusters behavior 
,
            ,
            ,
            ,
            Cluster EU star rating 
,
            ,
            ,
            ,
            Share of value added of cluster enterprises in GRP 
,
            ,
            ,
            ,
            Share of cluster products in the relevant world market segment  
,
            ,
            ,
            ,
            Share of export in cluster total volume of sales 
,
            ,
            ,
            ,
            Growth of the volume of production in the cluster companies 
,
            ,
            ,
            ,
             
,
            ,
            ,
            ,
            Growth of the volume of innovation production in the cluster 
,
            ,
            ,
            ,
            Share of export in cluster total volume of sales (by zones: US, EU, CIS, other countries)  
,
            ,
            ,
            ,
            Internal behavior 
,
            ,
            ,
            ,
            ,
            Median wage in the cluster 
,
            ,
            ,
            ,
            ,
            Growth of the volume of R&D in the cluster 
,
            ,
            ,
            ,
            ,
            Cluster collaboration 
,
            ,
            ,
            R&D 
,
            ,
            ,
            ,
            Patent map 
,
            ,
            ,
            ,
            Publications map 
,
            ,
            ,
            Industry 
,
            ,
            ,
            ,
            FDI map 
,
            ,
            ,
            ,
            Gazelle map 
,
            ,
            ,
            ,
            Business R&D expenditures as a share of revenues by sector 
,
            ,
            ,
            ,
            Share of regional products in the world market 
,
            ,
            ,
            ,
            Expenditure on innovation by firm size, by sector  
,
            ,
            ,
            Entrepreneurship 
,
            ,
            ,
            ,
            Startup map 
,
            ,
            ,
            ,
            Venture investment map 
,
            ,
            ,
            ,
            Attractiveness to public competitive funding 
,
            ,
            ,
            ,
            ,
            Fed and regional seed fund investments 
,
            ,
            ,
            ,
            ,
             
,
            ,
            Competitiveness support factors 
,
            ,
            ,
            Private investment in innovation 
